### ISSUE TITLE
ojsxc 2.1.0 in combination with "'asset-pipeline.enabled' => true" leads to garbled theme

### ISSUE THREAD

**Issue by DanScharon** [jsxc/jsxc#203]
When using ojsxc 2.1.0 in combination with concatenated/compressed CSS-/JS-files (see
https://doc.owncloud.org/server/8.0/admin_manual/configuration_server/js_css_asset_management_configuration.html?highlight=css ) the used theme is not getting loaded correctly but rather falls partly back to the default theme.

As a workaround setting asset-pipeline.enabled to false fixes the view. 

This does not occur with ojsxc 2.0.1

**Comment by sualko** [comment]
With the default theme it looks totally creepy:

![screen shot 2015-08-05 at 11 02 34](https://cloud.githubusercontent.com/assets/2974196/9082487/a8c2ea78-3b64-11e5-8879-13e13c2dfaaf.png)

Try to remove the following from <code>css/jsxc.oc.css</code>:

```
#jsxc_dialog {
  @-webkit-keyframes progress-bar-stripes {
    from {
      background-position: 40px 0; }

    to {
      background-position: 0 0; } }

  @keyframes progress-bar-stripes {
    from {
      background-position: 40px 0; }

    to {
      background-position: 0 0; } }
 }
```

This snippet causes the used asset library (https://github.com/kriswallsmith/assetic) to enclose multiple css definitions inside the <code>#jsxc_dialog</code> block. I will try to remove the above code with my build process.

**Comment by DanScharon** [comment]
yes, this did the trick, though I had to remove it in `apps/ojsxc/css/jsxc.oc.css` itself. Overlaying it within the theme as in `themes/themename/apps/ojsxc/css/jsxc.oc.css` didn't work.

**Comment by DanScharon** [comment]
unfortunately I cannot confirm that this is fixed in ojsxc 2.1.1
